test/test-call-gSNP/assert_vcf.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 function md5_vcf {
-    zcat $1 | grep -v '^##' | md5sum | cut -f 1 -d ' '
+    zcat -f $1 | grep -v '^##' | md5sum | cut -f 1 -d ' '
 }
 
 received=$(md5_vcf $1)
```

Comment:
Comment(non-blocking): following-up on above. additionally need double quotes here as well
```suggestion
received=$(md5_vcf "$1")
```